From c516b473fe84ab53f193fa0d46f39920404ceaf3 Mon Sep 17 00:00:00 2001
From: Poncho <poncho@spahan.ch>
Date: Fri, 21 Jul 2017 14:19:02 +0200
Subject: [PATCH] data/meson.build: fix Exec line in io.github.GnomeMpv.service

Without this patch, we get:
Exec=bin/gnome-mpv --gapplication-service

instead of
Exec=/usr/bin/gnome-mpv --gapplication-service
---
 data/meson.build | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index b859412..68d8909 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -31,8 +31,11 @@ install_data('gnome-mpv-symbolic.svg',
   install_dir: join_paths(icondir, 'symbolic/apps')
 )
 
+bindir = get_option ('bindir')
+prefix = get_option ('prefix')
+
 service_conf = configuration_data()
-service_conf.set('bindir', get_option('bindir'))
+service_conf.set('bindir', join_paths (prefix, bindir))
 
 configure_file(
   input: 'io.github.GnomeMpv.service.in',
-- 
2.13.0

